sled-agent/src/services.rs

```diff
@@ -694,12 +694,21 @@ impl ServiceManager {
                 ServiceType::ManagementGatewayService => {
                     info!(self.inner.log, "Setting up MGS service");
                     smfh.setprop("config/id", request.id)?;
+
+                    // Always tell MGS to listen on localhost so wicketd can
+                    // contact it even before we have an underlay network.
+                    smfh.addpropvalue(
+                        "config/address",
+                        &format!("[::1]:{MGS_PORT}"),
+                    )?;
+
                     if let Some(address) = request.addresses.get(0) {
-                        smfh.setprop(
+                        smfh.addpropvalue(
```

Comment:
Just to clarify, this second call to `addpropvalue` **appends** to the list of values, rather than replacing it, right?

Reply:
Correct! That is the primary difference between `setprop` and `addpropvalue`.